apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -785,6 +780,47 @@ class ScriptEditor extends React.Component {
           </div>
         </CollapsibleEditorSection>
 
+        <CollapsibleEditorSection title="Unit Calendar Settings">
```

Comment:
Do we want to hide this section for non-migrated scripts? It looks like these settings aren't used unless the script is migrated

Reply:
Great idea Bethany!